HISTORY.md

```diff
@@ -13,6 +13,7 @@
 * Accessing a partition of a partitioned filter or index through a pinned reference is no longer considered a cache hit.
 * The semantics of the per-block-type block read counts in the performance context now match those of the generic block_read_count.
 * Add C bindings for secondary instance, i.e. DBImplSecondary.
+* Rate limited deletion of WALs is only enabled if wal_dir == db_paths[0].path
```

Comment:
This description is confusing. Most users never set db_paths, it's RocksDB that fills db_paths[0].path with DB name. So to them it is empty.